frontend/src/component/common/StrategyItemContainer/StrategyItemContainer.tsx

```diff
@@ -44,15 +44,17 @@ const StyledCustomTitle = styled('div')(({ theme }) => ({
         display: 'block',
     },
 }));
-const StyledHeaderContainer = styled('div')({
-    flexDirection: 'column',
-    justifyContent: 'center',
-    verticalAlign: 'middle',
-});
+const StyledHeaderContainer = styled('hgroup')(({ theme }) => ({
+    display: 'flex',
+    flexFlow: 'row',
+    columnGap: '1ch',
+    fontSize: theme.typography.body1.fontSize,
+    '*:first-child': {
+        fontWeight: 'bold',
+    },
```

Comment:
We don't know whether the first item is going to be a header or a p element, but we always want the first item to be bold. We could also solve it by applying a class, but this feels easier.

Reply:
Update: use strategy name instead of first child.

Emotion complains that:
> The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".

So let's use a class instead.